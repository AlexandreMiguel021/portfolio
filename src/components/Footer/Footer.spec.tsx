import { render } from '@testing-library/react'
import { Footer } from '.'
import '@testing-library/jest-dom'

describe('Footer', () => {
	it('should render a Footer correctly', () => {
		const { container } = render(<Footer />)
		expect(container).toMatchSnapshot()
	})
})
