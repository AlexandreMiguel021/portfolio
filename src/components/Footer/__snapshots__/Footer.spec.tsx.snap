// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render a Footer correctly 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="wrapper"
    >
      <p>
        Â© 2022 Alexandre Miguel.
      </p>
      <div
        aria-hidden="false"
        class="icon"
        data-testid="backToTop"
        title="Voltar para o topo do site"
      >
        <svg
          fill="currentColor"
          height="1em"
          viewBox="0 0 16 16"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4.854 1.146a.5.5 0 0 0-.708 0l-4 4a.5.5 0 1 0 .708.708L4 2.707V12.5A2.5 2.5 0 0 0 6.5 15h8a.5.5 0 0 0 0-1h-8A1.5 1.5 0 0 1 5 12.5V2.707l3.146 3.147a.5.5 0 1 0 .708-.708l-4-4z"
            fill-rule="evenodd"
          />
        </svg>
      </div>
    </div>
  </footer>
</div>
`;
